packages/opentelemetry-core/src/context/propagation/HttpTraceContext.ts

```diff
@@ -73,7 +73,7 @@ function isValidSpanId(spanId: string): boolean {
  * Based on the Trace Context specification:
  * https://www.w3.org/TR/trace-context/
  */
-export class TraceContextFormat implements Propagator {
+export class HttpTraceContext implements HttpTextFormat {
```

Comment:
Is it important that the name is exactly the same? This is semantically incorrect. This is a propagator that propagates based on a specific format. It is not itself a format.

Reply:
Same for "context". This word is overloaded in so many places at this point that it has lost its meaning.

Reply:
> Is it important that the name is exactly the same?

This is just the first revision of APIs, things might change in the next revision. I would suggest to open an issue in [specifications](https://github.com/open-telemetry/opentelemetry-specification) repo. I think better ideas and approaches are always accepted.